bgc_data/bgc_chemistry.dataresource.yaml

```diff
@@ -21,10 +21,10 @@ schema:
   - name: SAMPLEDATELOCAL
     title: "Date & time of sample in local time (formatted string)"
     type: datetime
-    # format string *must* be specified using standard Python/C strptime specification         
-    # https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior               
+    # format string *must* be specified using standard Python/C strptime specification
+    # https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior
     format: "%Y-%m-%d %H:%M:%S"
-    required: false
+    required: true
```

Comment:
@clairedavies If I remember correctly you said in the catch-up meeting that these sample date fields should always be populated.
Same in all these tables?

Reply:
Yes, every sample has a date field